website/docs/d/catalog_item.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_catalog_item"
+sidebar_current: "docs-vcd-data-source-catalog-item"
+description: |-
+  Provides a catalog item data source.
+---
+
+# vcd\_catalog\_item
+
+Provides a vCloud Director Catalog item data source. A Catalog item can be used to reference a catalog item and use its 
+data within other resources or data sources.
+
+Supported in provider *v2.5+*
+
+## Example Usage
+
+```hcl
+data "vcd_catalog_item" "my-first-item" {
+  org     = "my-org"
+  catalog = "my-cat"
+  name    = "my-first-item"
+}
+
+resource "vcd_catalog_item" "my-second-item" {
+  org     = "${data.vcd_catalog_item.my-first-item.org}"
+  catalog = "${data.vcd_catalog_item.my-first-item.catalog}"
+
+  name                 = "my-second-item"
+  description          = "Belongs to ${data.vcd_catalog_item.my-first-item.name}"
+  ova_path             = "/path/to/test_vapp_template.ova"
```

Comment:
Interesting example. What does it do practically? Will it try to re-upload an OVA to an already existing item? I'd add a comment in the sample about what it does.

Reply:
every variable prepended by `data` will take the corresponding field from the data source and use it in the current field. 
For example the text `"Belongs to ${data.vcd_catalog_item.my-first-item.name}"` will become `"Belongs to my-first-item"`.

The data source is read-only. So it won't re-upload anything for the first item.

Reply:
comment added